src/git.rs

```diff
@@ -296,6 +298,28 @@ pub enum Ref {
     Commit(String),
 }
 
+impl Ref {
+    pub fn as_string(&self) -> String {
+        let (prefix, suffix) = match self {
+            Self::Branch(ref s) => ('b', s),
+            Self::Tag(ref s) => ('t', s),
+            Self::Commit(ref s) => ('c', s),
+        };
+
+        format!("{}{}", prefix, suffix)
+    }
```

Comment:
As we need this for file paths, we should sanitize it, for example:
- Replace `/` and `\` with `_`
- Only allow alphanumeric characters and some delimiters (`.,-_()#` for example) and remove the rest
- (Max length)

Reply:
Heh. But had the side effect (not sure if you noticed) that e.g. `release/v4.4` resulted in the creation of two sub-folders on the filesystem :p

Reply:
Yup exactly 😓